core/trino-main/src/test/java/io/trino/operator/scalar/TestStringFunctions.java

```diff
@@ -598,10 +599,11 @@ public void testSplitPart()
         assertInvalidFunction("SPLIT_PART(utf8(from_hex('CE')), '', 1)", "Invalid UTF-8 encoding");
     }
 
-    @Test(expectedExceptions = RuntimeException.class)
+    @Test
     public void testSplitPartInvalid()
     {
-        assertFunction("SPLIT_PART('abc-@-def-@-ghi', '-@-', 0)", VARCHAR, "");
+        assertThatThrownBy(() -> assertFunction("SPLIT_PART('abc-@-def-@-ghi', '-@-', 0)", VARCHAR, ""))
+                .isInstanceOf(RuntimeException.class);
```

Comment:
what is the message?
this probably should throw TrinoException and be tested with `assertInvalidFunction`

Reply:
Hm, it didn't have any message attached as such to the previous Test annotation. That is where I usallay look for the message a Test function is returning. Referring to: ` @Test(expectedExceptions = RuntimeException.class)`

The test ran successfully against the `RuntimeException.class`. I will replace it with  `TrinoException` and use `assertInvalidFunction. Will let you know the result.

Reply:
I re-wrote the test to following, and it works succesfully:

```    
@Test
    public void testSplitPartInvalid()
    {
       assertInvalidFunction("SPLIT_PART('abc-@-def-@-ghi', '-@-', 0)", "Index must be greater than zero");
    }
```
Lemme know if I should use this code. 

Also, `assertInvalidFunction()` does not seems to raise a throwable. If I wrap the method into `assertThatThrownBy()` as an instance of TrinoException or RuntimeException, the tests fails with the following error:  

```
 java.lang.AssertionError: 
Expecting code to raise a throwable.
```

Reply:
`assertInvalidFunction()` verifies `TrinoException` internally, so you don't need to wrap with `assertThatThrownBy()`. The example you posted looks fine to me. 
```java
    @Test
    public void testSplitPartInvalid()
    {
        assertInvalidFunction("SPLIT_PART('abc-@-def-@-ghi', '-@-', 0)", "Index must be greater than zero");
    }
```

Reply:
I see, thanks for confirming. I will commit new changes shortly.